airbyte-cdk/python/unit_tests/sources/declarative/requesters/paginators/test_offset_increment.py

```diff
@@ -6,18 +6,21 @@
 
 import pytest
 import requests
+
+from airbyte_cdk.sources.declarative.interpolation import InterpolatedString
 from airbyte_cdk.sources.declarative.requesters.paginators.strategies.offset_increment import OffsetIncrement
 
 
 @pytest.mark.parametrize(
     "test_name, page_size, expected_next_page_token, expected_offset",
     [
-        ("test_same_page_size", 2, 2, 2),
-        ("test_larger_page_size", 3, None, 0),
+        ("test_same_page_size", InterpolatedString(string="2", options={}), 2, 2),
```

Comment:
the page_size parameter can be an InterpolatedString, a str, and an int. we should have a test case for all 3 input types

Reply:
I made this fixes accordion to explanation to the comment above